src/Tidy/Operators/Defaults.purs

```diff
@@ -5,354 +5,5 @@ module Tidy.Operators.Defaults where
-  , """Data.Monoid.(<>) 5"""
-  , """Data.NaturalTransformation.(~>) type 4"""
-  , """Data.NonEmpty.(:|) 5"""
-  , """Data.Number.(%) 7"""
-  , """Data.Number.Approximate.(~=) 4"""
-  , """Data.Number.Approximate.(≅) 4"""
-  , """Data.Number.Approximate.(≇) 4"""
-  , """Data.Options.(:=) 6"""
-  , """Data.Ord.(<) 4"""
-  , """Data.Ord.(<=) 4"""
-  , """Data.Ord.(>) 4"""
-  , """Data.Ord.(>=) 4"""
-  , """Data.Profunctor.Choice.(+++) 2"""
-  , """Data.Profunctor.Choice.(|||) 2"""
-  , """Data.Profunctor.Strong.(&&&) 3"""
-  , """Data.Profunctor.Strong.(***) 3"""
-  , """Data.Ring.(*) 7"""
-  , """Data.Ring.(+) 6"""
-  , """Data.Ring.(-) 6"""
-  , """Data.Semigroup.(<>) 5"""
-  , """Data.Semiring.(*) 7"""
-  , """Data.Semiring.(+) 6"""
-  , """Data.Tuple.Nested.(/\) type 6"""
-  , """Data.Tuple.Nested.(/\) 6"""
-  , """Foreign.Index.(!) 9"""
-  , """Parsing.Combinators.(!!) 8"""
-  , """Parsing.Combinators.($>) 4"""
-  , """Parsing.Combinators.(..) 8"""
-  , """Parsing.Combinators.(:) 6"""
-  , """Parsing.Combinators.(<#>) 1"""
-  , """Parsing.Combinators.(<$) 4"""
-  , """Parsing.Combinators.(<$>) 4"""
-  , """Parsing.Combinators.(<?>) 4"""
-  , """Parsing.Combinators.(<??>) 3"""
-  , """Parsing.Combinators.(<@>) 4"""
-  , """Parsing.Combinators.(<|>) 3"""
-  , """Parsing.Combinators.(<~?>) 4"""
-  , """Parsing.Combinators.(\\) 5"""
-  , """Parsing.Indent.(<*/>) 11"""
-  , """Parsing.Indent.(<+/>) 9"""
-  , """Parsing.Indent.(<-/>) 10"""
-  , """Parsing.Indent.(<?/>) 12"""
-  , """Pathy.(<..>) 6"""
-  , """Pathy.(<.>) 6"""
-  , """Pathy.(</>) 6"""
-  , """Pathy.Path.(<..>) 6"""
-  , """Pathy.Path.(<.>) 6"""
-  , """Pathy.Path.(</>) 6"""
-  , """Prelude.(~>) type 4"""
-  , """Prelude.(#) 1"""
-  , """Prelude.($) 0"""
-  , """Prelude.($>) 4"""
-  , """Prelude.(&&) 3"""
-  , """Prelude.(*) 7"""
-  , """Prelude.(*>) 4"""
-  , """Prelude.(+) 6"""
-  , """Prelude.(-) 6"""
-  , """Prelude.(/) 7"""
-  , """Prelude.(/=) 4"""
-  , """Prelude.(<) 4"""
-  , """Prelude.(<#>) 1"""
-  , """Prelude.(<$) 4"""
-  , """Prelude.(<$>) 4"""
-  , """Prelude.(<*) 4"""
-  , """Prelude.(<*>) 4"""
-  , """Prelude.(<<<) 9"""
-  , """Prelude.(<=) 4"""
-  , """Prelude.(<=<) 1"""
-  , """Prelude.(<>) 5"""
-  , """Prelude.(<@>) 4"""
-  , """Prelude.(=<<) 1"""
-  , """Prelude.(==) 4"""
-  , """Prelude.(>) 4"""
-  , """Prelude.(>=) 4"""
-  , """Prelude.(>=>) 1"""
-  , """Prelude.(>>=) 1"""
-  , """Prelude.(>>>) 9"""
-  , """Prelude.(||) 2"""
-  , """StringParser.(<?>) 4"""
-  , """StringParser.Combinators.(<?>) 4"""
-  , """Test.QuickCheck.(/==) 2"""
-  , """Test.QuickCheck.(/=?) 2"""
-  , """Test.QuickCheck.(<=?) 2"""
-  , """Test.QuickCheck.(<?) 2"""
-  , """Test.QuickCheck.(<?>) 2"""
-  , """Test.QuickCheck.(===) 2"""
-  , """Test.QuickCheck.(==?) 2"""
-  , """Test.QuickCheck.(>=?) 2"""
-  , """Test.QuickCheck.(>?) 2"""
-  , """Type.Function.(#) type 1"""
-  , """Type.Function.($) type 0"""
-  , """Type.Prelude.(+) type 0"""
-  , """Type.Row.(+) type 0"""
+  [ """"""
```

Comment:
What happened here?

Reply:
Yikes! I didn't notice right away, but obviously generating the operators _isn't_ working. I'm debugging locally.

Reply:
I think it's failing because of removing `npm install` on my system, which also means no `spago install`, and therefore there's no `.spago` directory to read from when executing this in the tmpdir:

```
./bin/index.js generate-operators '.spago/*/*/src/**/*.purs'
```

Reply:
Fixed in 1cf24d1